lib/exception_notifier.rb

```diff
@@ -1,30 +1,66 @@
+# frozen_string_literal: true
+
 class ExceptionNotifier
-  def self.call(*args)
-    if exceptionable?(args.first)
-      capture_exception(args)
-    else
-      capture_message(args)
+  CONTEXT_TYPES = %i(extra_context tags_context user_context)
+  class << self
```

Comment:
I like this syntactic sugar, but it did result in an awful looking set of diffs.